kube/db-deployment.yaml

```diff
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2023 KernelCI Authors.
+# Author: Sanskar Bhushan <sbdtu5498@gmail.com>
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kernelci-api-db
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kernelci-api-db
+  template:
+    metadata:
+      labels:
+        app: kernelci-api-db
+    spec:
+      containers:
+        - name: kernelci-api-db
+          image: mongo:5.0
+          volumeMounts:
+            - name: mongo-data
+              mountPath: /data/db
+          # Resource limits needs to be discussed.
+          resources:
+  volumes:
+    - name: mongo-data
+      persistentVolumeClaim:
+        claimName: mongo-data-pvc
```

Comment:
Please fix the indentation here. `volumes` should be a sub-field of `spec`.

Reply:
Same for this one as well.

Reply:
Ah I see, thanks for noticing it. There was no YAML indenter so I missed it. Extremely sorry. I will fix it.